ethcore/src/client/test_client.rs

```diff
@@ -469,7 +469,7 @@ impl ChainInfo for TestBlockChainClient {
 impl BlockInfo for TestBlockChainClient {
 	fn block_header(&self, id: BlockId) -> Option<encoded::Header> {
 		self.block_hash(id)
-			.and_then(|hash| self.blocks.read().get(&hash).map(|r| Rlp::new(r).at(0).as_raw().to_vec()))
+			.and_then(|hash| self.blocks.read().get(&hash).map(|r| UntrustedRlp::new(r).at(0).unwrap().as_raw().to_vec()))
```

Comment:
We fought hard to get rid of `unwraps()` in the code base and this PR intorudces it back, can we please replace them with `expects()`?

Reply:
Yes sorry that unwrap snuck in. Removed.

Reply:
There seems to be way more than that one. It's fine in tests code, but would avoid it in regular code.

Reply:
I've removed the unwraps I added in this PR. There are plenty in there still from before - like you say probably okay for tests. Which is why I was lazy in this case and just added them. Broken windows and all!